open-metadata-implementation/governance-servers/open-lineage-services/open-lineage-services-server/pom.xml

```diff
@@ -27,6 +27,13 @@
     </properties>
     <dependencies>
 
+
+        <dependency>
+            <groupId>org.quartz-scheduler</groupId>
```

Comment:
This is introducing a significant new dependency. Can this be brought up on the next Egeria call to ensure the core maintainers are happy with the additional dependencyu?